src/domains/setting/i18n.ts

```diff
@@ -51,6 +51,14 @@ export const translations: { [key: string]: any } = {
 				TITLE: "Error Reporting",
 				DESCRIPTION: "Automatically send reports to help to improve our products.",
 			},
+			TRANSACTION_HISTORY: {
+				TITLE: "Portfolio Transaction History",
+				DESCRIPTION: "Enables transaction history on Portfolio page",
+			},
+			UPDATE_LEDGER: {
+				TITLE: "Update Ledger in Background",
+				DESCRIPTION: "Allows Ledger wallets to automatically update and sync data in the background.",
```

Comment:
why did this return 🤔

Reply:
:bleh:

Reply:
i think i moved those keys around before, so they didn't get purged with the merges. they are removed now.